src/content/team/team.json

```diff
@@ -73,6 +73,19 @@
         "email": "floody.o@northeastern.edu"
       }
     },
+    {
+      "name": "Zefeng (Daniel) Wang",
+      "team": {
+        "name": "Eboard",
+        "role": "Head of Booling | Dolphin of the East"
```

Comment:
what is `Booling`? dictionary does not recognize it as a word

Reply:
https://www.dictionary.com/e/slang/boolin/#:~:text=Boolin'%20means%20%E2%80%9Changing%20out%E2%80%9D,It%20comes%20from%20gang%20culture.

Reply:
that's not the right dictionary